src/adapterManager.js

```diff
@@ -207,6 +208,11 @@ function getAdUnitCopyForPrebidServer(adUnits, s2sConfig) {
 
   // don't send empty requests
   adUnitsCopy = adUnitsCopy.filter(adUnit => {
+    if (s2sConfig.filterBidderlessCalls) {
+      const onlyNull = adUnit.bids.length === 1 && adUnit.bids[0].bidder == null;
+      const hasStored = deepAccess(adUnit, 'ortb2Imp.ext.prebid.storedrequest.id') != null;
+      if (onlyNull && !hasStored) return false;
```

Comment:
```suggestion
      return !(adUnit.bids.length === 1 && adUnit.bids[0].bidder == null);
```
I think storedrequest is not the only way to have a valid bidderless request, so I'd just have this option filter out all ad units that have no bidders.